CONTRIBUTING.md

```diff
@@ -92,3 +92,16 @@ pip install -e .
 # run cou!
 cou --version
 ```
+
+## Testing the snap
+
+Installing a local snap has some manual steps because it must be installed in 'dangerous' mode:
```

Comment:
I think you can install with devmode, or not?

Reply:
You can install with devmode, but it's safer to install with `--dangerous`.

- `--dangerous`: installs the snap even if signatures from the snap store are missing
- `--devmode`: disables security sandbox, implicitly adds `--dangerous` too.

Reply:
So maybe this is explanation why the developer should test with `dangerous` mode? I feel like "must" mean they don't have other choices, but here I think we are recommending them to test with `dangerous` mode

Reply:
Yeah maybe the wording could be improved - what I mean there is that it cannot be installed normally - you must add the `--dangerous` flag (or of course `--devmode`).

Reply:
Thanks

Reply:
Thanks for your review :)